index.js

```diff
@@ -89,17 +158,25 @@ function session(options){
     , cookie = options.cookie || {}
     , trustProxy = options.proxy
     , storeReady = true
-    , rollingSessions = options.rolling || false;
+    , rollingSessions = options.rolling || false
+    , sessionId = options.sessionId || {};
+
+  sessionId.set = sessionId.set || exports.sessionId.set;
```

Comment:
We don't want to modify people's incoming objects (`sessionId` in this case); it brings lots of issues.